program/tests/helpers/create_mint_builder.rs

```diff
@@ -185,7 +184,7 @@ impl<'a> CreateMintBuilder<'a> {
             (wrapped_backpointer_address, wrapped_backpointer_account),
             (unwrapped_mint_addr, unwrapped_mint_account),
             (
-                system_program::id(),
+                solana_system_program::id(),
```

Comment:
Only if you don't need the `solana-system-program` crate for anything else:

```suggestion
                solana_sdk_ids::system_program::id(),
```